src/modules/WizardStep.jsx

```diff
@@ -889,6 +890,139 @@ const WizardStep = (props) => {
         },
 
         // V1.9.11 and later ends //////////////////////////////////////
+
+        
+        // V1.9.30 and later begins //////////////////////////////////////
+        {
+            title: 'Auto Polar Align',
+            label: 'Do you have the AutoPA add on:',
+            variable: 'autopa930',
+            condition: "$fwversion IN [V1930]",
+            preamble: ['////////////////////////////////', '// AutoPA Addon configuration ', '// Define whether we have the AutoPA add on or not. Currently: {v}'],
+            define: '',
+            control: {
+                type: 'radioimg',
+                choices: [
+                    { key: 'N', value: 'No AutoPA', image: '/images/none.png', additionalLines: ['// No AutoPA settings'] },
+                    { key: 'ALT', value: 'Altitude stepper only', image: '/images/autopa_alt.png' },
+                    { key: 'AZ', value: 'Azimuth stepper only', image: '/images/autopa_az.png' },
+                    { key: 'ALTAZ', value: 'Full AutoPA is installed', image: '/images/autopa.png' },
+                ]
+            },
+        },
+        {
+            title: 'AutoPA Version',
+            label: 'What version of AutoPA do you have installed:',
+            variable: 'autopaversion',
+            condition: "$autopa930 IN [AZ,ALT,ALTAZ]",
+            preamble: ['// Using AutoPA {v}.'],
+            define: '',
+            control: {
+                type: 'radioimg',
+                choices: [
+                    { key: '1', value: 'V1.0', image: '/images/none.png' },
+                    { key: '2', value: 'V2.0', image: '/images/none.png', additionalLines: ['#define AUTOPA_VERSION 2']  },
```

Comment:
Do you not want `#define AUTOPA 1` output for V1?

Reply:
No, the firmware assumes 1 unless told otherwise. I wanted to cut down on redundant config lines.